test/resources/api-1999-declaration-only-valid-example.json

```diff
@@ -0,0 +1,31 @@
+{
+  "reportDetails": {
+    "psrStatus": "Compiled",
+    "periodStart": "2023-04-06",
+    "periodEnd": "2024-04-05"
+  },
+  "accountingPeriodDetails": {
+    "accountingPeriods": [
+      {
+        "accPeriodStart": "2023-04-06",
+        "accPeriodEnd": "2024-04-05"
+      }
+    ]
+  },
+  "schemeDesignatory": {
+    "openBankAccount": "Yes",
+    "noOfActiveMembers": 23,
+    "noOfDeferredMembers": 45,
+    "noOfPensionerMembers": 6,
```

Comment:
In real life this can go through only if the total number of members is more than 100. Consider changing only if there are more changes to this PR, as it won't break or affect anything.